lib/models/fee_options/fee_option.dart

```diff
@@ -67,4 +70,9 @@ class RedeemOnchainFeeOption extends FeeOption {
     required super.processingSpeed,
     required super.satPerVbyte,
   });
+
+  @override
+  bool isAffordable({int? balance, int? walletBallance, required int amountSat}) {
```

Comment:
If `walletBallance` is required then we should check if not null and return false.

Reply:
Done.